### ISSUE TITLE
tty is not available. Docker for Mac setup has changed?

### ISSUE THREAD

**Issue by larsnystrom** [mariusgrigaitis/docker-mac-ssh-auth-sock#3]
I'm getting the message:
`/Users/lars/Library/Containers/com.docker.docker/Data/com.docker.driver.amd64-linux/tty is not available. Docker for Mac setup has changed?`

The contents of `~/Library/Containers/com.docker.docker/Data/` is:
```
$ ll ~/Library/Containers/com.docker.docker/Data/
total 0
drwxr-xr-x  12 lars  staff   384B 21 Sep 10:49 .
drwxr-xr-x@  3 lars  staff    96B 21 Sep 10:46 ..
srwxr-xr-x   1 lars  staff     0B 21 Sep 10:49 backend.sock
srwxr-xr-x   1 lars  staff     0B 21 Sep 10:49 docker.sock
srwxr-xr-x   1 lars  staff     0B 21 Sep 10:49 osxfs.sock
-rw-r--r--   1 lars  staff     0B 21 Sep 10:46 task.lock
drwxr-xr-x   6 lars  staff   192B 21 Sep 10:49 tasks
drwxr-xr-x   3 lars  staff    96B 21 Sep 10:46 vms
srwxr-xr-x   1 lars  staff     0B 21 Sep 10:49 vpnkit.diag.sock
srwxr-xr-x   1 lars  staff     0B 21 Sep 10:49 vpnkit.eth.sock
srwxr-xr-x   1 lars  staff     0B 21 Sep 10:49 vpnkit.pcap.sock
srwxr-xr-x   1 lars  staff     0B 21 Sep 10:49 vpnkit.port.sock
```

Using Docker for mac Version 18.06.1-ce-mac73 (26764).

**Comment by stijndm** [comment]
Seems it is now located at ~/Library/Containers/com.docker.docker/Data/vms/0/tty

**Comment by rfay** [comment]
There's a PR for this, https://github.com/mariusgrigaitis/docker-mac-ssh-auth-sock/pull/4

**Comment by mariusgrigaitis** [comment]
I seems I still have it with latest edge version (Version 2.0.0.0-mac82 (29268))

```
lrwxr-xr-x   1 mgrigaitis  staff     5B May 17  2018 com.docker.driver.amd64-linux -> vms/0
```

Not sure why it's missing in other versions

**Comment by mariusgrigaitis** [comment]
Solved with #4
